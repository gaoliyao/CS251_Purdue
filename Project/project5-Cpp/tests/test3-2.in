3
100 600
AAA AAP 1.00
AAB AAK 2.00
AAC AAZ 3.00
AAD ADG 4.00
AAE ACX 5.00
AAF AAY 6.00
AAG ADB 7.00
AAH AAL 8.00
AAI ACK 9.00
AAJ ABM 10.00
AAK AAP 11.00
AAL ACH 12.00
AAM ABI 13.00
AAN AAB 14.00
AAO ADJ 15.00
AAP ACD 16.00
AAQ ABB 17.00
AAR AAG 18.00
AAS AAX 19.00
AAT ABW 20.00
AAU AAW 21.00
AAV ACY 22.00
AAW ADD 23.00
AAX ACI 24.00
AAY ADB 25.00
AAZ AAK 26.00
ABA AAH 27.00
ABB AAV 28.00
ABC ACS 29.00
ABD ACE 30.00
ABE AAM 31.00
ABF ABL 32.00
ABG ACO 33.00
ABH ABL 34.00
ABI AAV 35.00
ABJ ADQ 36.00
ABK AAN 37.00
ABL ACA 38.00
ABM ACF 39.00
ABN ACX 40.00
ABO ADM 41.00
ABP ACU 42.00
ABQ ABI 43.00
ABR ACZ 44.00
ABS AAZ 45.00
ABT ACW 46.00
ABU ABG 47.00
ABV AAE 48.00
ABW ABG 49.00
ABX AAH 50.00
ABY ACA 51.00
ABZ ACD 52.00
ACA ABK 53.00
ACB ABI 54.00
ACC ACP 55.00
ACD ADS 56.00
ACE ACW 57.00
ACF ABK 58.00
ACG AAS 59.00
ACH ABE 60.00
ACI AAA 61.00
ACJ ACD 62.00
ACK ABX 63.00
ACL ABL 64.00
ACM ABC 65.00
ACN ABR 66.00
ACO AAD 67.00
ACP ABH 68.00
ACQ ACA 69.00
ACR ABE 70.00
ACS AAX 71.00
ACT ACY 72.00
ACU AAR 73.00
ACV AAA 74.00
ACW AAC 75.00
ACX ADN 76.00
ACY ADG 77.00
ACZ AAG 78.00
ADA AAX 79.00
ADB ADO 80.00
ADC ACH 81.00
ADD ABE 82.00
ADE ABC 83.00
ADF ABT 84.00
ADG ADT 85.00
ADH ADR 86.00
ADI ADP 87.00
ADJ ACU 88.00
ADK ADG 89.00
ADL ACM 90.00
ADM ACC 91.00
ADN ADG 92.00
ADO ACT 93.00
ADP AAD 94.00
ADQ ACW 95.00
ADR AAA 96.00
ADS ADU 97.00
ADT ACZ 98.00
ADU ABH 99.00
ADV ABY 100.00
AAH AAI 101.00
ABB AAY 102.00
ACJ ADD 103.00
AAP ADT 104.00
ABN ADN 105.00
ABP ABY 106.00
ABV AAT 107.00
AAR ABP 108.00
ADN AAB 109.00
AAF ABT 110.00
ADH ABC 111.00
AAB AAL 112.00
ABC ADV 113.00
ADK AAN 114.00
AAC ABW 115.00
ACW ADD 116.00
AAO ABO 117.00
ABK ACB 118.00
ADF ADA 119.00
ADV AAD 120.00
AAX ACQ 121.00
ABS ACO 122.00
AAV AAB 123.00
ACM ACH 124.00
ABE ACN 125.00
ACS AAK 126.00
AAQ ACH 127.00
ACY ACS 128.00
AAH AAC 129.00
AAD ABF 130.00
AAX ACZ 131.00
ACT AAG 132.00
AAH ABB 133.00
ABE ADI 134.00
AAF ABX 135.00
ADC ACR 136.00
ADI ADA 137.00
AAZ ACC 138.00
ADL AAB 139.00
AAZ ABW 140.00
ACD ABC 141.00
ADC ADS 142.00
ADU ACD 143.00
AAZ ACR 144.00
ACK ABH 145.00
ADO ABE 146.00
ACL AAT 147.00
AAJ ACQ 148.00
AAU ADM 149.00
ADL ABC 150.00
ACH ABD 151.00
ADE ABW 152.00
ABE AAH 153.00
ABX ABL 154.00
ADJ ABD 155.00
ABH ADH 156.00
ABK ACH 157.00
AAG ADU 158.00
ADO AAG 159.00
ACX ABK 160.00
ADL ABN 161.00
AAH ABC 162.00
ABO ADT 163.00
ACE ABT 164.00
AAY ACP 165.00
ACW AAG 166.00
ACQ ACF 167.00
ACN AAR 168.00
ADQ ACB 169.00
ADU ADC 170.00
ADM ABI 171.00
ABN ABX 172.00
ABH ADF 173.00
ACD ACI 174.00
AAT ADS 175.00
ADV ABA 176.00
ADR AAH 177.00
ACC ADJ 178.00
ACE ACL 179.00
ABG ADC 180.00
ADJ ABY 181.00
AAQ AAE 182.00
ACP AAK 183.00
ACF ACN 184.00
ABQ AAA 185.00
AAA ADD 186.00
ABX ADH 187.00
AAQ ACE 188.00
ABT ADJ 189.00
ACB ABT 190.00
ACO AAA 191.00
AAE AAU 192.00
ADJ ACI 193.00
ABJ ACU 194.00
ADO AAS 195.00
ACQ ABV 196.00
AAJ ABT 197.00
AAB ACA 198.00
ADH ABU 199.00
ABM ACC 200.00
ACD ABK 201.00
ABP ACI 202.00
ABH ACH 203.00
ACI ADL 204.00
ADC ADV 205.00
ABX ACP 206.00
ACT ADP 207.00
ADH AAC 208.00
AAZ AAG 209.00
ABX AAN 210.00
ABQ AAL 211.00
ACG ADR 212.00
ABX AAA 213.00
AAC ABN 214.00
ACC AAJ 215.00
ABB ABW 216.00
ACR ACI 217.00
AAH ADD 218.00
ADD ABY 219.00
AAH ACA 220.00
ADS ABS 221.00
AAH ACV 222.00
ABY AAI 223.00
ADI ABS 224.00
AAT ABS 225.00
ABO ADE 226.00
AAA ABP 227.00
ABJ ACD 228.00
ABG ADL 229.00
ACM ACI 230.00
ABL ABI 231.00
ACU ABT 232.00
AAP ACW 233.00
ABK ABW 234.00
ACY ABR 235.00
ACB ACU 236.00
ABO ACI 237.00
ABT ADM 238.00
ACQ ABF 239.00
ABJ ADK 240.00
ABC ABB 241.00
AAW ADC 242.00
AAU ACF 243.00
ABJ ACA 244.00
ADV AAA 245.00
ACM ABK 246.00
ADI ADL 247.00
ABH ACB 248.00
AAP ACS 249.00
AAC ADN 250.00
ACN AAH 251.00
ACM AAF 252.00
ACP ACJ 253.00
ADS ABJ 254.00
ACX ACV 255.00
ACK ABY 256.00
AAF ADE 257.00
AAH ABP 258.00
ADI AAG 259.00
ADP ABZ 260.00
ADR ABF 261.00
ADO ADC 262.00
ADG AAH 263.00
ABY ABM 264.00
ADU AAQ 265.00
ABT AAO 266.00
AAV ACM 267.00
ABC ACR 268.00
AAE ABB 269.00
ADU ACO 270.00
ACZ AAD 271.00
ABW ABL 272.00
ADS ADJ 273.00
ADR ADL 274.00
ABM ADM 275.00
AAU ADE 276.00
ACT ACF 277.00
ADL ACV 278.00
ADR ABN 279.00
ADL ADP 280.00
ACC ACL 281.00
ACF ADE 282.00
ADG AAJ 283.00
AAH ADL 284.00
ABL ACF 285.00
AAH ACO 286.00
AAM AAI 287.00
AAD AAJ 288.00
ADR ADV 289.00
ADU ADH 290.00
ABP ACT 291.00
ACP AAM 292.00
ABC ACE 293.00
ADI ACX 294.00
ADR ABB 295.00
ACQ AAB 296.00
ABQ ADA 297.00
ABJ ABB 298.00
ADJ ABQ 299.00
ACQ ACY 300.00
ABZ ACX 301.00
ABR ACM 302.00
ABJ ABU 303.00
ACV ABE 304.00
ADT AAX 305.00
AAP ABN 306.00
ABU ADE 307.00
AAD ABA 308.00
ADM ABP 309.00
AAC ABM 310.00
AAV AAW 311.00
ABN ACL 312.00
ACA ACX 313.00
ABQ ABO 314.00
ACR AAK 315.00
AAQ ACV 316.00
ABM AAL 317.00
ABL AAZ 318.00
AAK ACK 319.00
ACE AAH 320.00
ABL AAX 321.00
ADU ABK 322.00
ACG ACC 323.00
ACK ABW 324.00
ADR AAQ 325.00
ABM AAQ 326.00
ABN ADA 327.00
ABG ADN 328.00
AAF ACW 329.00
ADF ABA 330.00
ADV ABB 331.00
ACL ADK 332.00
ACA ACV 333.00
AAC ACI 334.00
ABH ABO 335.00
ADG ADF 336.00
ACY ADQ 337.00
ABL ADM 338.00
ADQ ABH 339.00
AAH ABH 340.00
AAB ADU 341.00
ACL ADH 342.00
ABP ACQ 343.00
AAM AAZ 344.00
ABU AAB 345.00
ACZ ADU 346.00
ABC ADO 347.00
ADI ABG 348.00
AAS ADL 349.00
ADD ABD 350.00
ADI ACF 351.00
ACX AAY 352.00
ABV ACR 353.00
AAC ACK 354.00
AAE AAR 355.00
AAA ABU 356.00
ABL AAM 357.00
ACT ADG 358.00
AAN AAA 359.00
ADE ADP 360.00
ABS AAU 361.00
AAZ ACK 362.00
AAJ ACS 363.00
AAR ADM 364.00
ABZ AAE 365.00
ADS ABP 366.00
AAF ABY 367.00
ACD AAK 368.00
ACQ ACD 369.00
ACE ACF 370.00
ACP ADB 371.00
ABP ADC 372.00
ADB ACX 373.00
ACV AAX 374.00
ADS ABZ 375.00
AAV ACD 376.00
AAL ADV 377.00
ABZ ABR 378.00
ABY ABV 379.00
ADG ACE 380.00
ABY ABO 381.00
ACO ACS 382.00
ABV ACW 383.00
ABB AAP 384.00
ACB AAV 385.00
ABV ADG 386.00
ABW ACV 387.00
ACH ABS 388.00
ACY ADT 389.00
ACC ADT 390.00
AAF AAG 391.00
AAW AAQ 392.00
ACV ADS 393.00
ACM ACX 394.00
ADK ADE 395.00
ABT ABK 396.00
AAI AAZ 397.00
AAD ACJ 398.00
ABU ABY 399.00
ADT ADQ 400.00
ACX ACE 401.00
ADN ABZ 402.00
AAG ADT 403.00
AAD AAX 404.00
ADB ACI 405.00
ABX ADN 406.00
ACQ ACX 407.00
ABO ADG 408.00
AAC ABC 409.00
AAS AAA 410.00
ACM ABA 411.00
AAZ ACP 412.00
ABN AAX 413.00
ACS ABK 414.00
ACR ABT 415.00
ABT AAM 416.00
ABX ABZ 417.00
AAJ ABY 418.00
ACK ACM 419.00
AAZ ADP 420.00
AAZ ABB 421.00
ABL ABT 422.00
ACC ACZ 423.00
ADE ACE 424.00
ACF AAA 425.00
ADB ABH 426.00
ADP AAS 427.00
AAI ACL 428.00
AAH ADA 429.00
ACJ ACA 430.00
ABQ AAK 431.00
AAD AAE 432.00
AAM AAR 433.00
ACQ ABL 434.00
ACK ABT 435.00
AAQ ADV 436.00
ADN ACS 437.00
ABC AAZ 438.00
ADB ADI 439.00
AAZ ADJ 440.00
ACI AAE 441.00
ACV ACB 442.00
ACX ADD 443.00
ACR ABI 444.00
AAL ADE 445.00
ADI ACC 446.00
ABS ABP 447.00
ACG ACE 448.00
ACG ADA 449.00
ABT AAU 450.00
AAY AAO 451.00
ACU ACP 452.00
ADG AAA 453.00
ADO AAP 454.00
ADI ACR 455.00
AAD ADU 456.00
ABA ABC 457.00
AAE ACB 458.00
AAJ ACV 459.00
ADJ AAV 460.00
ACD ACV 461.00
ABB ABZ 462.00
ACO ABL 463.00
ACH AAY 464.00
AAP AAE 465.00
ADS ADN 466.00
ACS ACQ 467.00
AAK AAH 468.00
ACR ACC 469.00
AAW AAH 470.00
AAY ACZ 471.00
AAG AAC 472.00
AAF AAK 473.00
ACD ACP 474.00
ABJ ABQ 475.00
ADP ABH 476.00
ACE ACA 477.00
AAJ ACT 478.00
AAI ACF 479.00
AAP ADB 480.00
ABA AAZ 481.00
ADI ABV 482.00
ADC ACI 483.00
ACC ACE 484.00
ABM ACI 485.00
AAW ACN 486.00
ACK ACF 487.00
ACH AAC 488.00
AAV ADO 489.00
AAL ACZ 490.00
ADS ACU 491.00
AAB AAF 492.00
ABD AAQ 493.00
ADP AAW 494.00
ADE AAJ 495.00
ABH ACU 496.00
AAV ADN 497.00
ACQ ABS 498.00
AAI ABE 499.00
ACB ACP 500.00
ADH ACB 501.00
ABT AAG 502.00
ADT AAI 503.00
ABK ABU 504.00
ADC ABL 505.00
AAD AAK 506.00
AAF ABN 507.00
AAR ABY 508.00
AAN ACU 509.00
ACY ABV 510.00
ABH AAJ 511.00
AAU AAG 512.00
ACB ABO 513.00
AAC ACJ 514.00
AAW ACE 515.00
ACJ ABA 516.00
ACO AAK 517.00
ABJ AAC 518.00
ACE ACP 519.00
ADN ACH 520.00
ACZ ADS 521.00
ABU AAP 522.00
ACT ACW 523.00
ABK ABJ 524.00
ACK ABP 525.00
ACX ACN 526.00
ACC ABX 527.00
AAV ABJ 528.00
ACF ABI 529.00
ACK ACX 530.00
ABS ABX 531.00
ACZ ACB 532.00
AAQ ACQ 533.00
ACM ABU 534.00
AAQ AAP 535.00
ADP AAO 536.00
ADE AAQ 537.00
ABP ABT 538.00
ACU ACZ 539.00
ADC ABI 540.00
AAS AAH 541.00
ABZ AAY 542.00
ACF AAY 543.00
ACI ACO 544.00
ACG ACW 545.00
ABP ACD 546.00
AAI ADN 547.00
ADM AAY 548.00
ABL ACG 549.00
ADN ADE 550.00
ACU ACW 551.00
ABY ACN 552.00
AAT AAW 553.00
ABQ ACA 554.00
AAJ AAM 555.00
AAL AAM 556.00
ABL AAU 557.00
ADI ADR 558.00
AAX ADB 559.00
AAC ADU 560.00
ABP ABR 561.00
ADJ ADB 562.00
AAB ADA 563.00
AAN ABA 564.00
ACA ACM 565.00
ADN AAY 566.00
ABM ADI 567.00
ACY ADV 568.00
ADU ABN 569.00
AAM ADJ 570.00
ACI ABX 571.00
ADK ABU 572.00
ABS AAL 573.00
ADA ADV 574.00
ACJ ABF 575.00
ACX ADS 576.00
ABI ABC 577.00
ABB ABV 578.00
ACC ADB 579.00
ACL ADU 580.00
AAD AAC 581.00
ADG ABF 582.00
AAB ABI 583.00
ACT AAN 584.00
ACW ADF 585.00
ABA ACH 586.00
ACG ADJ 587.00
ABR AAT 588.00
ABQ AAS 589.00
ABU ADR 590.00
ACY ACB 591.00
ABA ABN 592.00
ABZ ADE 593.00
ABP ABJ 594.00
ACN ADR 595.00
AAV ABK 596.00
ACI ADR 597.00
AAT AAX 598.00
ACG AAB 599.00
ABI ADG 600.00
AAM
