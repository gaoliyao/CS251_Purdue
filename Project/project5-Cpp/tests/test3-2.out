AAM
ABE
ACH
AAL
AAH
ABA
ABX
ACK
AAI
ADC
ACR
ADD
AAW
AAU
ACJ
ACD
AAP
AAA
ACI
AAX
AAS
ACG
ACS
ABC
ACM
ADL
ADE
ADA
ACV
ADR
ADH
AAK
AAB
AAN
ABK
ACA
ABL
ABF
ABH
ACP
ACC
ADM
ABO
ACL
ABY
ADV
ACQ
ACF
ABM
AAJ
AAZ
AAC
ACW
ABT
ADF
ACE
ABD
ADQ
ABJ
ABS
ABZ
ADS
ADU
ABI
AAV
ABB
AAQ
ACY
ACT
ADO
ADB
AAG
AAR
ACU
ABP
ADJ
AAO
ACZ
ABR
ACN
AAY
AAF
ADG
AAD
ACO
ABG
ABU
ABW
AAT
ADP
ADI
ADK
ADN
ACX
AAE
ABV
ABN
ADT
ABQ
ACB
